src/components/data/LocalVariablesContent.js

```diff
@@ -285,7 +285,9 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
       }
     }),
     deleteIndex !== undefined && h(Modal, {
-      onDismiss: setDeleteIndex,
+      onDismiss: () => {
+        setDeleteIndex(undefined)
+      },
```

Comment:
This is so simple that I would still try to constrain the vertical space used to express it, i.e.,
```suggestion
      onDismiss: () => { setDeleteIndex(undefined) },
```
Feel free to ignore if the linter complains.